### DISCUSSION TITLE
Livewire 3 wire:navigate return blank page with text "2023"

### DISCUSSION THREAD

**Discussion by LaurentOuattara** [livewire/livewire#7018] [1. Help]
I am developing an application with Laravel Livewire 3 and a bootstrap template
I created a layout as indicated, for now I have created two components Dashbord and Calendar
When I navigate between the two pages using wire:navigate, I receive a blank page with the text "2023". But when I refresh, the real page is displayed.
I don't know what the problem is, please help me

**Comment by sajjadhossainshohag** [comment]
Check your footer section, year print by JavaScript.  I hope it works by removing the script tag line.

**Reply by LaurentOuattara** [reply]
it works thank you

**Reply by DevIP98** [reply]
Thanks.

**Reply by hellodit** [reply]
Please elaborate your answer

**Reply by aqsajimmy** [reply]
@sajjadhossainshohagbd holly god, YOU ARE FCKKKN GENIUS , ITS FCKKN WORRKSS
THANK YOU , NEVER THINK TO REMOVE THAT LITTLE SHTTTTT T_T
im cryinnn
thanks

**Reply by mhdeveloperYs** [reply]
Thank you, bro, its working
